analyzers/src/SonarAnalyzer.CSharp/Wrappers/ObjectCreationFactory.cs

```diff
@@ -44,6 +44,11 @@ node switch
                 _ => throw new InvalidOperationException("Unexpected type: " + node.GetType().Name)
             };
 
+        public static IObjectCreation TryCreate(SyntaxNode node) =>
```

Comment:
Look for references to the old "Create" call and see if you can replace some calls with this simplified version (clean as you code).

Reply:
Oh sorry, I thought you suggested doing it on a different PR, I'll do it as part of this one then.